common/src/main/java/com/lightstep/tracer/shared/SpanBuilder.java

```diff
@@ -20,7 +20,7 @@
     static final String PARENT_SPAN_GUID_KEY = "parent_span_guid";
 
     private final String operationName;
-    private Long spanId = null;
+    private String spanId = null;
```

Comment:
This is going to be problematic as the client migrates to the GRPC protocol and its `int64` span and trace ids. I'm ok with that for our purposes here (which are prototype-like) but wanted to mention it as an FYI.